lib/Dialect/FIRRTL/Transforms/ExpandWhens.cpp

```diff
@@ -298,15 +314,15 @@ class LastConnectResolver : public FIRRTLVisitor<ConcreteT> {
   }
 
   void visitStmt(ConnectOp op) {
-    setLastConnect(getFieldRefFromValue(op.getDest()), op);
+    setConnect(getFieldRefFromValue(op.getDest()), op);
```

Comment:
It seems a good rename.

Reply:
(might be a better name, mostly dropping "Last" part to reflect it handles more than those...)

Reply:
Going with `recordConnect`, FWIW.